posthog/models/raw_sessions/sql.py

```diff
@@ -0,0 +1,523 @@
+from django.conf import settings
+
+from posthog.clickhouse.table_engines import (
+    Distributed,
+    ReplicationScheme,
+    AggregatingMergeTree,
+)
+
+TABLE_BASE_NAME = "raw_sessions"
+RAW_SESSIONS_DATA_TABLE = lambda: f"sharded_{TABLE_BASE_NAME}"
+
+TRUNCATE_RAW_SESSIONS_TABLE_SQL = (
+    lambda: f"TRUNCATE TABLE IF EXISTS {RAW_SESSIONS_DATA_TABLE()} ON CLUSTER '{settings.CLICKHOUSE_CLUSTER}'"
```

Comment:
Heads up! This will work for now, but in the future something to know:

If there is any prod data in this table this will fail due to `max_table_size_to_drop` setting
https://clickhouse.com/docs/en/operations/server-configuration-parameters/settings#max_table_size_to_drop

We use this to protect from accidental deletions. Protects us from `DROP`s and `TRUNCATES`

Reply:
Good to know, the delete/truncates are only used in test setup code so should be ok